date_range/models/date_range.py

```diff
@@ -22,7 +22,7 @@ def _default_company(self):
         ondelete='restrict', domain="['|', ('company_id', '=', company_id), "
                                     "('company_id', '=', False)]")
     type_name = fields.Char(
-        string='Type', related='type_id.name', readonly=True, store=True)
+        related='type_id.name', readonly=True, store=True, string="Type Name")
```

Comment:
@astirpe  you must remove the string since "Type Name" is the default string for a field named type_name.

Reply:
I have put it precisely because now Odoo checks for the existence of 2 labels with the same name, so you have "Name" and "Name", and this avoids the warning.

Reply:
@pedrobaeza the conflicts was on string='Type', not on Name. Removing all the string="..." fixes the warning to me.

Reply:
Oh and... now I know who is the one that is changing the code :laughing: @pedrobaeza

Reply:
Yes, I have squashed a bit the commit history and fixed that problem for moving forward with this. I hope you don't bother for that.

Reply:
@pedrobaeza no problem of course! I'm going to remove string="Type Name" anyway. Do you have objections?

Reply:
@pedrobaeza PLZ don't squash the history on module where you are not the maintainer. :smirk:

Reply:
@lmignon Pedro was right, we need to write string="Type Name" explicitly, otherwise runbot gives the following warning:

WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (type_name, name) of date.range() have the same label: Name.